src/Skoruba.IdentityServer4.Admin.EntityFramework.Shared/Configuration/DatabaseProviderConfiguration.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Skoruba.IdentityServer4.Admin.EntityFramework.Shared.Configuration
+{
+    public class DatabaseProviderConfiguration
+    {
+        public string ProviderType { get; set; }
```

Comment:
Is it valid to use the enum type here or does this have to be a string in this scenario?

Reply:
Done, good idea 👍